packages/cli-repl/package.json

```diff
@@ -115,8 +115,8 @@
   "optionalDependencies": {
     "get-console-process-list": "^1.0.4",
     "glibc-version": "^1.0.0",
-    "macos-export-certificate-and-key": "^1.1.1",
-    "mongodb-crypt-library-version": "^1.0.3",
-    "win-export-certificate-and-key": "^1.1.1"
+    "macos-export-certificate-and-key": "^1.1.2",
```

Comment:
Was almost driven insane by the "fact" that this bump of the library made CI fail even though no code changed (it surely can't be a coincidence, right, this PR bumps the package that provides the error message that's leading to the CI errors??), then finally thought to check other runs on `main`, turns out the CI redness happens completely independently of this PR without any code changes at all: https://github.com/mongodb-js/mongosh/pull/1991 🫠